samples/python/tutorial_code/features2D/Homography/panorama_stitching_rotating_camera.py

```diff
@@ -0,0 +1,54 @@
+import cv2 as cv
+import numpy as np
+
+
```

Comment:
Remove extra line feed?

There are also some extra line feeds that could be stripped below and in the other sample code.